### PR TITLE
Add script to check SDL resends OnSeekMediaClockTimer afrom all HMI l…

### PR DESCRIPTION
**Pull request by OVikhrov**
…evels if it allowed by Policy
@aarotar Please review

### CODE REVIEW

File: test_scripts/API/OnSeekMediaClockTimer/ATF_P_OnSeekMediaClockTimer_Policy_Allowed_In_All_HMI_Levels.lua [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+-- In case
+-- SDL receives OnSeekMediaClockTimer notification from HMI
+-- and this notification is valid and allowed by Policies
+-- SDL must:
+-- transfer OnSeekMediaClockTimer notification from HMI to mobile app
+--
+-- 1. Used preconditions:
+-- a) First SDL life cycle
+-- b) OnSeekMediaClockTimer notification allowed in preloaded file for default app for all HMI levels
+-- c) App successfylly registered, consented and activated
```
Comment (aarotar) [inline test_scripts/API/OnSeekMediaClockTimer/ATF_P_OnSeekMediaClockTimer_Policy_Allowed_In_All_HMI_Levels.lua]: @OVikhrov, please check syntax in script

File: test_scripts/API/OnSeekMediaClockTimer/ATF_P_OnSeekMediaClockTimer_Policy_Allowed_In_All_HMI_Levels.lua [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+--
+-- 2. Performed steps:
+-- a) HMI sends OnSeekMediaClockTimer notification to SDL in all HMI levels.
+--
+-- Expected result:
+-- a) SDL resends OnSeekMediaClockTimer notification to mobile app for all HMI levels.
+---------------------------------------------------------------------------------------------
+
```
Comment (aarotar) [inline test_scripts/API/OnSeekMediaClockTimer/ATF_P_OnSeekMediaClockTimer_Policy_Allowed_In_All_HMI_Levels.lua]: @OVikhrov, plase avoid redundant empty lines

File: test_scripts/API/OnSeekMediaClockTimer/ATF_P_OnSeekMediaClockTimer_Policy_Allowed_In_All_HMI_Levels.lua (hunk 1 above)
Comment (OVikhrov) [inline test_scripts/API/OnSeekMediaClockTimer/ATF_P_OnSeekMediaClockTimer_Policy_Allowed_In_All_HMI_Levels.lua]: Accepted

File: test_scripts/API/OnSeekMediaClockTimer/ATF_P_OnSeekMediaClockTimer_Policy_Allowed_In_All_HMI_Levels.lua (hunk 2 above)
Comment (OVikhrov) [inline test_scripts/API/OnSeekMediaClockTimer/ATF_P_OnSeekMediaClockTimer_Policy_Allowed_In_All_HMI_Levels.lua]: Accepted

**Comment by OVikhrov** [comment]
@aarotar Plese review iteration#2

**Pull request merged**